### DISCUSSION TITLE
Reset password no longer working and returning a 403 error

### DISCUSSION THREAD

**Discussion by joaoguidev** [supabase/supabase#27327] [Questions]
Hello

More recently I noticed that my app's reset password is not working anymore. Returning me a 403 error and the log on Supabase is:

**"msg": "403: Session from session_id claim in JWT does not exist"**

Below is the reset password code been executed:
```
const { data, error } = await supabase.auth.updateUser({
  password: 'new password'
})
```

How do I solve that?

**Comment by GaryAustin1** [comment]
https://github.com/supabase/auth/issues/1579

**Comment by silentworks** [comment]
For now you will need to redirect the user to a sign in page if they have updated their password since the session is revoked after a password update. I have left a comment on the issue Gary linked above to ask the Supabase team to document this new behavior and make an announcement of this breaking change.
